presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestRaptorSplitManager.java

```diff
@@ -215,7 +217,8 @@ private void deleteShardNodes()
     private static ConnectorSplitSource getSplits(RaptorSplitManager splitManager, ConnectorTableLayoutResult layout)
     {
         ConnectorTransactionHandle transaction = new RaptorTransactionHandle();
-        return splitManager.getSplits(transaction, SESSION, layout.getTableLayout().getHandle(), new SplitSchedulingContext(UNGROUPED_SCHEDULING, true));
+        List<PrestoWarning> warningList = new ArrayList<PrestoWarning>();
```

Comment:
We don't usually use list to denote a collector. We use explicit collectors. Objects in our codebase are mostly immutable whenever necessary.